assemblies/assembly-rhdh-telemetry.adoc

```diff
@@ -0,0 +1,33 @@
+[id="assembly-rhdh-telemetry_{context}"]
+= Telemetry data collection
+
+The telemetry data collection feature is enabled by default. {product} sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin.
```

Comment:
I don't think we should have this on by default until we have a documented approval from Legal. https://issues.redhat.com/browse/RHIDP-58?focusedId=24806334&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24806334 contains no such documentation.